command/asset/example.nomad.hcl

```
@@ -23,10 +22,9 @@ job "example" {
 
   # The "type" parameter controls the type of job, which impacts the scheduler's
   # decision on placement. This configuration is optional and defaults to
-  # "service". For a full list of job types and their differences, please see
-  # the online documentation.
+  # "service". For a full list of job types and their differences, please see:
```

```suggestion
  # "service".
```
I moved "full list..." to L27 since the "refer to" link is repeated there.